crates/nu-engine/src/command_args.rs

```diff
@@ -237,6 +237,15 @@ impl EvaluatedCommandArgs {
         }
     }
 
+    // If switch is present returns true, otherwise false
+    // Returns error if flag with name `name` has a value and the value is not convertible to bool
+    pub fn get_switch(&self, name: &str) -> Result<bool, ShellError> {
```

Comment:
We have this already with `has_flag`. You could call that instead of making a new function.

Reply:
Oops. How could I miss `has_flag` :]. Changed this ofc.